spec/controllers/api/rest/admin/cdr/cdr_exports_controller_spec.rb

```diff
@@ -86,7 +86,7 @@
         expect(response.status).to eq(422)
         expect(JSON.parse(response.body)['errors']).to match_array(
           hash_including(
-            'detail' => 'filters - unknown_filter not allowed'
+            'detail' => 'filters - unknown-filter not allowed'
```

Comment:
@gigorok  why this introduced on rails 4-5 migration commit? do not understand how is related

Reply:
this change is related with upgrading of `jsonapi-resources` from 0.9.1.beta1 to 0.9.1.beta2. looks like this PR https://github.com/cerebris/jsonapi-resources/pull/1069

Reply:
@gigorok  thanks, good to know,  jsonapi-resources update via separate commit would be perfect

Reply:
@Fivell I was wrong. nothing should be changed in this case. 0.9.1.beta2 deal with filters on index action, we are performing custom validation error. anyway I rolled back jsonapi-resources to 0.9.1.beta1